cli/smartcontract/contract_test.go

```diff
@@ -833,6 +856,21 @@ func TestComlileAndInvokeFunction(t *testing.T) {
 				})
 			})
 		})
+
+		t.Run("with await", func(t *testing.T) {
+			cmd = []string{"neo-go", "contract", "invokefunction",
+				"--rpc-endpoint", "http://" + e.RPC.Addresses()[0],
+				"--out", txout,
+				"--wallet", testcli.ValidatorWallet, "--address", testcli.ValidatorAddr,
+				"--force",
+				"--await",
+			}
+			e.In.WriteString("one\r")
+			e.Run(t, append(cmd, h.StringLE(), "getValue")...)
+			line := e.GetNextLine(t)
+			_, err := util.Uint256DecodeStringLE(line)
+			require.NoError(t, err)
```

Comment:
NACK, use `CheckAwaitableTxPersisted` here and everywhere where `--await` flag specified in test.